FreeRTOS/Demo/RISC-V-Qemu-virt_GCC/ns16550.c

```diff
@@ -20,7 +20,7 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
  * https://www.FreeRTOS.org
- * https://www.github.com/FreeRTOS
+ * https://github.com/FreeRTOS
```

Comment:
Just to be curious, why remove this and why keep it on www. FreeRTOS ?

Reply:
To satisfy the header check - the aim to ensure consistency across files.